src/Modules.js

```diff
@@ -2,49 +2,19 @@
 * Core modules and their info files
 */
 module.exports = {
-  'channel': [
-    require('./modules/Channel/Module'), require('./modules/Channel/info')
-  ],
-  'brightness': [
-    require('./modules/Brightness/Module'), require('./modules/Brightness/info')
-  ],
-  'edge-detect': [
-    require('./modules/EdgeDetect/Module'), require('./modules/EdgeDetect/info')
-  ],
-  'ndvi': [
-    require('./modules/Ndvi/Module'), require('./modules/Ndvi/info')
-  ],
-  'invert': [
-    require('./modules/Invert/Module'), require('./modules/Invert/info')
-  ],
-  'crop': [
-    require('./modules/Crop/Module'), require('./modules/Crop/info')
-  ],
-  'colormap': [
-    require('./modules/Colormap/Module'), require('./modules/Colormap/info')
-  ],
-  'decode-qr': [
-    require('./modules/DecodeQr/Module'), require('./modules/DecodeQr/info')
-  ],
-  'fisheye-gl': [
-    require('./modules/FisheyeGl/Module'), require('./modules/FisheyeGl/info')
-  ],
-  'dynamic': [
-    require('./modules/Dynamic/Module'), require('./modules/Dynamic/info')
-  ],
-  'blur': [
-    require('./modules/Blur/Module'), require('./modules/Blur/info')
-  ],
-  'saturation': [
-    require('./modules/Saturation/Module'), require('./modules/Saturation/info')
-  ],
-  'average': [
-    require('./modules/Average/Module'), require('./modules/Average/info')
-  ],
-  'blend': [
-    require('./modules/Blend/Module'), require('./modules/Blend/info')
-  ],
-  'import-image': [
-    require('./modules/ImportImage/Module'), require('./modules/ImportImage/info')
-  ]
+  'channel': require('./modules/Channel'),
+  'brightness': require('./modules/Brightness'),
+  'edge-detect': require('./modules/EdgeDetect'),
+  'ndvi': require('./modules/Ndvi'),
+  'crop': require('./modules/Crop'),
+  'colormap': require('./modules/Colormap'),
+  'decode-qr': require('./modules/DecodeQr'),
+  'fisheye-gl': require('./modules/FisheyeGl'),
+  'dynamic': require('./modules/Dynamic'),
+  'blur': require('./modules/Blur'),
+  'saturation': require('./modules/Saturation'),
+  'average': require('./modules/Average'),
+  'blend': require('./modules/Blend'),
+  'import-image': require('./modules/ImportImage')
```

Comment:
Could we even just iterate through all modules in `./modules/_____` and rely on their info to set the name?

Reply:
@jywarren that would be tough since we do have to require them somewhere in order to iterate over them, right?